src/app/personside/infotabs/meldinger/traadliste/TraadListe.tsx

```diff
@@ -83,16 +62,19 @@ function TraadListe(props: Props) {
                 />
             </InputStyle>
             <SokVerktøyStyle>
-                {traadKomponenter.length !== props.traader.length ? (
-                    <Element>
-                        Søket traff {traadKomponenter.length} av {props.traader.length} tråder
-                    </Element>
-                ) : (
-                    <Element>Totalt {props.traader.length} tråder</Element>
-                )}
-                <EkspanderKnapp onClick={() => props.setSokeord('')} tittel={'Nullstill søk'} />
+                <Normaltekst>{soketreffTekst}</Normaltekst>
+                {props.sokeord !== '' && <LenkeKnapp onClick={() => props.setSokeord('')}>Vis alle tråder</LenkeKnapp>}
```

Comment:
Nå skriver vi tråder to steder og "Søk i meldinger". Skal vi holde oss til ett ord eller er det riktig at vi søker i meldinger, men vi viser tråder?
<img width="342" alt="Screen Shot 2019-09-27 at 1 08 20 PM" src="https://user-images.githubusercontent.com/11627538/65765249-0926f400-e128-11e9-876d-da8501bd4804.png">

Reply:
Jepp, tror du kan endre til meldinger intill videre, vi fikk tilbakemelding på at tråd ikke var et begrep de kjente til